### ISSUE TITLE
[Feature Request] Force 8 decimal places on amount and fee values

### ISSUE THREAD

**Issue by ottosch** [sparrowwallet/sparrow#1547]
In some places, decimal places are trimmed if the last digits are zero. I think this can lead the user to send the wrong amount and is inconsistent with some other places.

For example, the "balance" and "value" columns of "Transactions/UTXOs" grid always display 8 decimal places, but the "send" button and "amount" and "fee" inputs of the send view don't.

![utxos](https://github.com/user-attachments/assets/3d008947-e611-4ceb-aa95-d8ace289f632)
![send](https://github.com/user-attachments/assets/0c6ddcb2-5a1a-4dce-a4db-638772dcbba4)

I noticed this can be adjusted with `setMinimumFractionDigits(8)` on `UnitFormat::getBtcFormat()` for the send button, although "Mempool" label will display "0.00000000 BTC" instead of "0 BTC" (which I'm ok with, but might be undesirable) and on `CoinFilter::coinFormat` for the text fields.

**Comment by craigraw** [comment]
I don't think this is a good idea in this case. A UI should always aim to display the less surprising result, and in general a textfield should just display the text entered into it wherever possible. It's possible to add some small changes (like thousands separators) but adding 8 or 9 characters into the textfield on the first digit entry seems too much to me. There are plenty of existing places to check the amount entered while constructing and signing the transaction.
